tests/e2e.sh

```diff
@@ -67,13 +67,14 @@ touch "${HOME}"/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
 
-if [[ "$MINIKUBE_DRIVER" != "none" ]]; then 
-  export MINIKUBE_PROFILE_ARG=" --profile ${MINIKUBE_PROFILE}"
+if [[ "$MINIKUBE_DRIVER" != "none" ]]; then
```

Comment:
Let's set the default value to the variable first. This eliminates the need for the else statement.

Once the setting is done, we can then `export` the var as opposed to `export`ing in every conditional path